examples/resources/images/license.md

```diff
@@ -102,6 +102,14 @@ https://opengameart.org/content/super-mario-world-style-tiles
 CC0 1.0
 ```
 
+## blend/dest.png
```

Comment:
What about source.png?

Reply:
It's fully made by me.

Here is sources with both examples:
![blendmodes](https://user-images.githubusercontent.com/19890545/216920471-75447e18-fe5d-4355-a440-380710ff5f6e.svg)